src/ui/settingsdialog.cpp

```diff
@@ -68,6 +68,24 @@ const char DocsetNameProperty[] = "docsetName";
 const char DownloadTypeProperty[] = "downloadType";
 const char DownloadPreviousReceived[] = "downloadPreviousReceived";
 const char ListItemIndexProperty[] = "listItem";
+
+QString timeSinceLastUpdate(const QDateTime &updateTime)
+{
+    int diff = updateTime.secsTo(QDateTime::currentDateTime());
+
+    if (diff <= 5)      return QStringLiteral("just now");
+    if (diff <= 20)     return QStringLiteral("%1 seconds ago").arg(diff);
+    if (diff < 60)      return QStringLiteral("less than a minute ago");
+    if (diff <= 90)     return QStringLiteral("one minute ago");
+    if (diff <= 3540)   return QStringLiteral("%1 minutes ago").arg(std::round(diff / 60));
+    if (diff <= 5400)   return QStringLiteral("1 hour ago");
```

Comment:
Why `one minute ago`, but `1 hour ago`? I think texts should be consistent between units.